internal/handlers/pg/msg_aggregate.go

```diff
@@ -436,15 +436,15 @@ func processStagesStats(ctx context.Context, closer *iterator.MultiCloser, p *st
 				"count", collStats.CountObjects,
 				"avgObjSize", avgObjSize,
 				"storageSize", collStats.SizeCollection,
-				"freeStorageSize", int64(0), // TODO https://github.com/FerretDB/FerretDB/issues/2342
-				"capped", false, // TODO https://github.com/FerretDB/FerretDB/issues/2342
-				"wiredTiger", must.NotFail(types.NewDocument()), // TODO https://github.com/FerretDB/FerretDB/issues/2342
+				"freeStorageSize", int64(0),
```

Comment:
The task https://github.com/FerretDB/FerretDB/issues/2342 is closed, so I guess we don't need to keep this TODO in the old backend.

Reply:
Thanks :pray: